src/test/rustdoc/type-layout.rs

```diff
@@ -52,3 +52,12 @@ pub struct Unsized([u8]);
 
 // @!has type_layout/trait.MyTrait.html 'Size: '
 pub trait MyTrait {}
+
+// @has type_layout/enum.Variants.html 'Size: '
+// @has - '2 bytes'
+// @has - '<code>A</code>: 0 bytes'
+// @has - '<code>B</code>: 1 byte'
```

Comment:
Just to confirm: this checks that this "text" is in the raw HTML directly, right?

Reply:
What's odd is I tried `@has - '<strong>Size:</strong> 8 bytes` on master on an existing test case, but it failed. I'm not sure why these tests are passing.